addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1214,6 +1213,16 @@ const UnitUserValueWidget = UserValueWidget.extend({
     _floatToStr: function (value) {
         return `${parseFloat(value.toFixed(5))}`;
     },
+    /**
+     * Checks whether the widget has a unit.
+     *
+     * @private
+     * @returns {Boolean} true if it has a unit, false otherwise.
```

Comment:
**an** unit (multiple times + in the commit message)

Reply:
No because the "u" is pronounced with a "y" (like "you-nit") so in that case we have to use "a" (like a university vs an umbrella) :slightly_smiling_face: